packages/odyssey-react/src/components/Toast/Toast.module.scss

```diff
@@ -62,7 +62,7 @@
 
 .icon {
   width: var(--IconSize);
-  color: var(--IconColor);
+  color: var(--IconTextColor);
```

Comment:
wouldn't this just be `IconColor` in the same way we use `font-size` declarations but create tokens name `IconSize` ? Perhaps you can check and see what patterns are more prevalent in the code base for us now

Reply:
This is the only place that we set it specifically, with color passing linting seems safe to use the more condensed version.

Reply:
addressed in 671cd3a